src/app/modules/item/http-services/update-item-string.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { assertSuccess, SimpleActionResponse } from '../../../shared/http-services/action-response';
+import { environment } from '../../../../environments/environment';
+import { map } from 'rxjs/operators';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UpdateItemStringService {
+
+  constructor(private http: HttpClient) {
+  }
+
+  updateItem(
+    itemId: string,
+    changes: object,
```

Comment:
why not have 
`{
description: string|null,
image_url: string|null,
subtitle: string|null,
title: string|null
}` instead of `object` here ?

Reply:
Yes, true. "object" type should not be used in Typescript. (we could use {string -> unknown} dictionary at least.)
The problem is that we should not repeat all attributes in each class and here it does not really help preventing  future problems as its output data. 
But at least for item strings where the supported attribute are not many, we should add them.

Note that all these attributes should be optional in addition to be nullable.